apps/docs/pages/guides/database/extensions/wrappers/overview.mdx

```diff
@@ -1,74 +1,121 @@
-select
-  customer_id
-  currency
-from
-   stripe.customers;
+-- Copy data from your production database to your
+-- data warehouse for the last 24 hours:
 
-    customer_id     | currency 
---------------------+-----------
- cus_MJiBtCqOF1Bb3F | usd      
-(1 row)
+insert into warehouse.analytics
+select * from public.analytics
+where ts > (now() - interval '1 DAY');
 ```
 
-## Usage
+This approach provides several benefits:
 
-### Enable the extension
+1. **Simplicity:** the Wrappers API is just SQL, so data engineers don't need to learn new tools and languages.
+1. **Save on time:** avoid setting up additional data pipelines.
+1. **Save on Data Engineering costs:** less infrastructure to be managed.
 
-<Tabs
-  scrollable
-  size="small"
-  type="underlined"
-  defaultActiveId="dashboard"
-  queryGroup="database-method"
->
-<TabPanel id="dashboard" label="Dashboard">
+On the flip-side, Wrappers are not as feature-rich as these tools, and they couple the ETL process to your database.
 
-1. Go to the [Database](https://supabase.com/dashboard/project/_/database/tables) page in the Dashboard.
-2. Click on **Extensions** in the sidebar.
-3. Search for "wrappers" and enable the extension.
+### On-demand ETL with Wrappers
 
-</TabPanel>
-<TabPanel id="sql" label="SQL">
+Supabase extends the ETL concept with real-time data access. Instead of moving gigabytes of data from one system to another before you can query it, you can instead query the data directly from the remote server. This additional option, "Query", extends the ETL process and is called [QETL](https://www.sciencedirect.com/science/article/abs/pii/S0169023X1730438X) (pronounced "kettle"): Query, Extract, Transform, Load.
 
 {/* prettier-ignore */}
 ```sql
--- Enable the "wrappers" extension
-create extension wrappers;
+-- Get all purchases for a user from your data warehouse:
+select
+  auth.users.id as user_id,
+  warehouse.orders.id as order_id
+from
+  warehouse.orders
+join 
+  auth.users on auth.users.id = warehouse.orders.user_id
+where 
+  auth.users.id = '<some_user_id>';
+```
+
+This approach has several benefits:
+
+1. **On-demand:** analytical data is immediately available within your application with no additional infrastructure.
+1. **Always in sync:** since the data is queried directly from the remote server, it's always up-to-date.
+1. **Integrated:** large datasets are available within your application, and can be joined with your operational/transactional data.
+1. **Save on bandwidth:** only extract/load what you need.
 
--- Disable the "wrappers" extension
-drop extension if exists wrappers;
+### Batch ETL with Wrappers
+
+A common use case for Wrappers is to extract data from a production database and load it into a data warehouse. This can be done within your database using [pg_cron](/docs/guides/database/extensions/pgcron). For example, you can schedule a job to run every night to extract data from your production database and load it into your data warehouse.
```

Comment:
```suggestion
A common use case for Wrappers is to extract data from a production database and load it into a data warehouse. This can be done within your database using [pg_cron](/docs/guides/database/extensions/pg_cron). For example, you can schedule a job to run every night to extract data from your production database and load it into your data warehouse.
```